src/cdk/drag-drop/directives/drag.spec.ts

```diff
@@ -5712,6 +5757,21 @@ class StandaloneDraggable {
   freeDragPosition?: {x: number, y: number};
 }
 
+@Component({
+  template: `
+    <div class="wrapper" style="width: 200px; height: 200px; background: green;">
+      <div
+        cdkDrag
+        #dragElement
+        style="transform: none; width: 100px; height: 100px; background: red;"></div>
+    </div>
+  `
+})
+class StandaloneDraggableWithInitialTransformNone {
```

Comment:
Rather than creating a new fixture, you should be able to reuse the `StandaloneDraggable` and adjust the test. E.g. something like:

```
const fixture = createComponent(StandaloneDraggable);
fixture.detectChanges();
const dragElement = fixture.componentInstance.dragElement.nativeElement;

dragElement.style.transform = 'none';
dragElementViaMouse(fixture, dragElement, 50, 100);
expect(dragElement.style.transform).toBe('translate3d(50px, 100px, 0px)');
```